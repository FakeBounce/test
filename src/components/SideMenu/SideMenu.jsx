import React, { Component } from "react";
import styles from "./SideMenu.module.scss";
import { Nav, NavItem, Media } from "reactstrap";
import { NavLink } from "react-router-dom";
import Logo from "assets/images/sidemenu/atome.svg";
import { translate } from "common/methods/translations";

import { PAPREC_SITE } from "common/consts/common.js";

class SideMenu extends Component {
  render() {
    const currentLanguage = localStorage.getItem("language");

    return (
      <div className={styles.SideMenuContainer}>
        <a href={PAPREC_SITE + currentLanguage} target="_blank">
          <Media object src={Logo} className={styles.Logo} />
        </a>
        <Nav className={styles.Nav}>
          <NavItem className={styles.NavItem}>
            <NavLink
              href={"/home"}
              to={"/home"}
              activeClassName={styles._active}
            >
              <div className={styles.Home} />
              {translate("globals.menu.home").toUpperCase()}
            </NavLink>
          </NavItem>
          <NavItem className={styles.NavItem}>
            <NavLink
              href={"/myContracts"}
              to={"/myContracts"}
              activeClassName={styles._active}
            >
              <div className={styles.Contracts} />
              {translate("globals.menu.myContracts").toUpperCase()}
            </NavLink>
          </NavItem>
          <NavItem className={styles.NavItem}>
            <NavLink
              href={"/myFiles"}
              to={"/myFiles"}
              activeClassName={styles._active}
            >
              <div className={styles.Files} />
              {translate("globals.menu.myFiles").toUpperCase()}
            </NavLink>
          </NavItem>
          <NavItem className={styles.NavItem}>
            <NavLink
              href={"/myRemovals"}
              to={"/myRemovals"}
              activeClassName={styles._active}
            >
              <div className={styles.Truck} />
              {translate("globals.menu.myRemovals").toUpperCase()}
            </NavLink>
          </NavItem>
          <NavItem className={styles.NavItem}>
            <NavLink
              href={"/myReports"}
              to={"/myReports"}
              activeClassName={styles._active}
            >
              <div className={styles.Report} />
              {translate("globals.menu.myReports").toUpperCase()}
            </NavLink>
          </NavItem>
          <NavItem className={styles.NavItem}>
            <NavLink
              href={"/myCollaborativeSpace"}
              to={"/myCollaborativeSpace"}
              activeClassName={styles._active}
            >
              <div className={styles.Collab} />
              {translate("globals.menu.myCollaborativeSpace").toUpperCase()}
            </NavLink>
          </NavItem>
        </Nav>
      </div>
    );
  }
}
export default SideMenu;
