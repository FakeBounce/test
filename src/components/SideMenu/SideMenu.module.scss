@import '~common/styles/variables';
.SideMenuContainer {
  width: $metrics-regular;
  position: fixed;
  top: 0;
  left: 0;
  bottom: 0;
  //background: radial-gradient(circle, rgba(82,94,194,0.9) 0%, #2F3976 100%);
  Background: linear-gradient(#2f3976, rgba(55, 65, 135, 0.98) 50%, rgba(82, 94, 194, 0.9));
  box-shadow: inset 0 1px 3px 0 rgba(0,0,0,0.5), 0 2px 4px 0 rgba(0,0,0,0.5);
  overflow: hidden;
  z-index: 10;
  &:hover {
    width: 200px;
  }
  .Logo {
    margin: 7px;
    display: block;
    max-width: 46px;
    height: 46px;
  }
  .Nav {
    display: flex;
    flex-direction: column;
    .NavItem {
      height: $metrics-regular;
      display: flex;
      justify-content: flex-start;
      &:hover {
        transition: background-color .3s;
        background-color: $blue;
      }
      &:first-child {
        border-top: 1px solid $white;
      }
      ._active {
        background-color : $blue; 
      }
      .Home, .Contracts, .Contracts, .Files, .Truck, .Report, .Collab{
        height: 23px;
        min-width: 38px;
        margin: 20px 11px;
        display: block;
      }
      > a {
        display: flex;
        height: $metrics-regular;
        width: 100%;
        align-items: center;
        color: $white;
        text-decoration: none;
        font-size: 12px;
        text-align: left;
        padding: 0;
      }
    }
  }
}

.Home {
  background: url($path-images +"sidemenu/home_icon.svg") no-repeat center;
}
.Contracts {
  background: url($path-images +"sidemenu/contracts_icon.svg") no-repeat center;
}
.Files {
  background: url($path-images +"sidemenu/files_icon.svg") no-repeat center;
}
.Truck {
  background: url($path-images +"sidemenu/truck_icon.svg") no-repeat center;
}
.Report {
  background: url($path-images +"sidemenu/report_icon.svg") no-repeat center;
}
.Collab {
  background: url($path-images +"sidemenu/collab_icon.svg") no-repeat center;
}