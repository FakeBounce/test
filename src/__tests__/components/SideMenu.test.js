import React from "react";
import { shallow, configure } from "enzyme/build";
import Adapter from "enzyme-adapter-react-16/build";
import SideMenu from "components/SideMenu/SideMenu";

configure({ adapter: new Adapter() });

describe("Testing SideMenu component", () => {
  it("renders as expected", () => {
    const wrapper = shallow(<SideMenu />);
    expect(wrapper).toMatchSnapshot();
  });
});
