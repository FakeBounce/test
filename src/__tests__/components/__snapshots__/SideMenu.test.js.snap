// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing SideMenu component renders as expected 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <SideMenu />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <a
          href="https://www.paprec.com/undefined"
          target="_blank"
>
          <Media
                    className={undefined}
                    object={true}
                    src="atome.svg"
          />
</a>,
        <Nav
          className={undefined}
          tag="ul"
          vertical={false}
>
          <NavItem
                    className={undefined}
                    tag="li"
          >
                    <NavLink
                              activeClassName="active"
                              aria-current="page"
                              href="/home"
                              to="/home"
                    >
                              <div
                                        className={undefined}
                              />
                              ACCUEIL
                    </NavLink>
          </NavItem>
          <NavItem
                    className={undefined}
                    tag="li"
          >
                    <NavLink
                              activeClassName="active"
                              aria-current="page"
                              href="/myContracts"
                              to="/myContracts"
                    >
                              <div
                                        className={undefined}
                              />
                              MES CONTRATS
                    </NavLink>
          </NavItem>
          <NavItem
                    className={undefined}
                    tag="li"
          >
                    <NavLink
                              activeClassName="active"
                              aria-current="page"
                              href="/myFiles"
                              to="/myFiles"
                    >
                              <div
                                        className={undefined}
                              />
                              MES DOCUMENTS 
                               EN LIGNE
                    </NavLink>
          </NavItem>
          <NavItem
                    className={undefined}
                    tag="li"
          >
                    <NavLink
                              activeClassName="active"
                              aria-current="page"
                              href="/myRemovals"
                              to="/myRemovals"
                    >
                              <div
                                        className={undefined}
                              />
                              MES ENLÈVEMENTS
                    </NavLink>
          </NavItem>
          <NavItem
                    className={undefined}
                    tag="li"
          >
                    <NavLink
                              activeClassName="active"
                              aria-current="page"
                              href="/myReports"
                              to="/myReports"
                    >
                              <div
                                        className={undefined}
                              />
                              MES REPORTINGS
                    </NavLink>
          </NavItem>
          <NavItem
                    className={undefined}
                    tag="li"
          >
                    <NavLink
                              activeClassName="active"
                              aria-current="page"
                              href="/myCollaborativeSpace"
                              to="/myCollaborativeSpace"
                    >
                              <div
                                        className={undefined}
                              />
                              MON ESPACE COLLABORATIF
                    </NavLink>
          </NavItem>
</Nav>,
      ],
      "className": undefined,
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <Media
            className={undefined}
            object={true}
            src="atome.svg"
/>,
          "href": "https://www.paprec.com/undefined",
          "target": "_blank",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "className": undefined,
            "object": true,
            "src": "atome.svg",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        "type": "a",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "children": Array [
            <NavItem
              className={undefined}
              tag="li"
>
              <NavLink
                            activeClassName="active"
                            aria-current="page"
                            href="/home"
                            to="/home"
              >
                            <div
                                          className={undefined}
                            />
                            ACCUEIL
              </NavLink>
</NavItem>,
            <NavItem
              className={undefined}
              tag="li"
>
              <NavLink
                            activeClassName="active"
                            aria-current="page"
                            href="/myContracts"
                            to="/myContracts"
              >
                            <div
                                          className={undefined}
                            />
                            MES CONTRATS
              </NavLink>
</NavItem>,
            <NavItem
              className={undefined}
              tag="li"
>
              <NavLink
                            activeClassName="active"
                            aria-current="page"
                            href="/myFiles"
                            to="/myFiles"
              >
                            <div
                                          className={undefined}
                            />
                            MES DOCUMENTS 
                             EN LIGNE
              </NavLink>
</NavItem>,
            <NavItem
              className={undefined}
              tag="li"
>
              <NavLink
                            activeClassName="active"
                            aria-current="page"
                            href="/myRemovals"
                            to="/myRemovals"
              >
                            <div
                                          className={undefined}
                            />
                            MES ENLÈVEMENTS
              </NavLink>
</NavItem>,
            <NavItem
              className={undefined}
              tag="li"
>
              <NavLink
                            activeClassName="active"
                            aria-current="page"
                            href="/myReports"
                            to="/myReports"
              >
                            <div
                                          className={undefined}
                            />
                            MES REPORTINGS
              </NavLink>
</NavItem>,
            <NavItem
              className={undefined}
              tag="li"
>
              <NavLink
                            activeClassName="active"
                            aria-current="page"
                            href="/myCollaborativeSpace"
                            to="/myCollaborativeSpace"
              >
                            <div
                                          className={undefined}
                            />
                            MON ESPACE COLLABORATIF
              </NavLink>
</NavItem>,
          ],
          "className": undefined,
          "tag": "ul",
          "vertical": false,
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "children": <NavLink
                activeClassName="active"
                aria-current="page"
                href="/home"
                to="/home"
>
                <div
                                className={undefined}
                />
                ACCUEIL
</NavLink>,
              "className": undefined,
              "tag": "li",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "activeClassName": "active",
                "aria-current": "page",
                "children": Array [
                  <div
                    className={undefined}
/>,
                  "ACCUEIL",
                ],
                "href": "/home",
                "to": "/home",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "className": undefined,
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "div",
                },
                "ACCUEIL",
              ],
              "type": [Function],
            },
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "children": <NavLink
                activeClassName="active"
                aria-current="page"
                href="/myContracts"
                to="/myContracts"
>
                <div
                                className={undefined}
                />
                MES CONTRATS
</NavLink>,
              "className": undefined,
              "tag": "li",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "activeClassName": "active",
                "aria-current": "page",
                "children": Array [
                  <div
                    className={undefined}
/>,
                  "MES CONTRATS",
                ],
                "href": "/myContracts",
                "to": "/myContracts",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "className": undefined,
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "div",
                },
                "MES CONTRATS",
              ],
              "type": [Function],
            },
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "children": <NavLink
                activeClassName="active"
                aria-current="page"
                href="/myFiles"
                to="/myFiles"
>
                <div
                                className={undefined}
                />
                MES DOCUMENTS 
                 EN LIGNE
</NavLink>,
              "className": undefined,
              "tag": "li",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "activeClassName": "active",
                "aria-current": "page",
                "children": Array [
                  <div
                    className={undefined}
/>,
                  "MES DOCUMENTS 
 EN LIGNE",
                ],
                "href": "/myFiles",
                "to": "/myFiles",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "className": undefined,
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "div",
                },
                "MES DOCUMENTS 
 EN LIGNE",
              ],
              "type": [Function],
            },
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "children": <NavLink
                activeClassName="active"
                aria-current="page"
                href="/myRemovals"
                to="/myRemovals"
>
                <div
                                className={undefined}
                />
                MES ENLÈVEMENTS
</NavLink>,
              "className": undefined,
              "tag": "li",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "activeClassName": "active",
                "aria-current": "page",
                "children": Array [
                  <div
                    className={undefined}
/>,
                  "MES ENLÈVEMENTS",
                ],
                "href": "/myRemovals",
                "to": "/myRemovals",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "className": undefined,
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "div",
                },
                "MES ENLÈVEMENTS",
              ],
              "type": [Function],
            },
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "children": <NavLink
                activeClassName="active"
                aria-current="page"
                href="/myReports"
                to="/myReports"
>
                <div
                                className={undefined}
                />
                MES REPORTINGS
</NavLink>,
              "className": undefined,
              "tag": "li",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "activeClassName": "active",
                "aria-current": "page",
                "children": Array [
                  <div
                    className={undefined}
/>,
                  "MES REPORTINGS",
                ],
                "href": "/myReports",
                "to": "/myReports",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "className": undefined,
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "div",
                },
                "MES REPORTINGS",
              ],
              "type": [Function],
            },
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "children": <NavLink
                activeClassName="active"
                aria-current="page"
                href="/myCollaborativeSpace"
                to="/myCollaborativeSpace"
>
                <div
                                className={undefined}
                />
                MON ESPACE COLLABORATIF
</NavLink>,
              "className": undefined,
              "tag": "li",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "activeClassName": "active",
                "aria-current": "page",
                "children": Array [
                  <div
                    className={undefined}
/>,
                  "MON ESPACE COLLABORATIF",
                ],
                "href": "/myCollaborativeSpace",
                "to": "/myCollaborativeSpace",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "className": undefined,
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "div",
                },
                "MON ESPACE COLLABORATIF",
              ],
              "type": [Function],
            },
            "type": [Function],
          },
        ],
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <a
            href="https://www.paprec.com/undefined"
            target="_blank"
>
            <Media
                        className={undefined}
                        object={true}
                        src="atome.svg"
            />
</a>,
          <Nav
            className={undefined}
            tag="ul"
            vertical={false}
>
            <NavItem
                        className={undefined}
                        tag="li"
            >
                        <NavLink
                                    activeClassName="active"
                                    aria-current="page"
                                    href="/home"
                                    to="/home"
                        >
                                    <div
                                                className={undefined}
                                    />
                                    ACCUEIL
                        </NavLink>
            </NavItem>
            <NavItem
                        className={undefined}
                        tag="li"
            >
                        <NavLink
                                    activeClassName="active"
                                    aria-current="page"
                                    href="/myContracts"
                                    to="/myContracts"
                        >
                                    <div
                                                className={undefined}
                                    />
                                    MES CONTRATS
                        </NavLink>
            </NavItem>
            <NavItem
                        className={undefined}
                        tag="li"
            >
                        <NavLink
                                    activeClassName="active"
                                    aria-current="page"
                                    href="/myFiles"
                                    to="/myFiles"
                        >
                                    <div
                                                className={undefined}
                                    />
                                    MES DOCUMENTS 
                                     EN LIGNE
                        </NavLink>
            </NavItem>
            <NavItem
                        className={undefined}
                        tag="li"
            >
                        <NavLink
                                    activeClassName="active"
                                    aria-current="page"
                                    href="/myRemovals"
                                    to="/myRemovals"
                        >
                                    <div
                                                className={undefined}
                                    />
                                    MES ENLÈVEMENTS
                        </NavLink>
            </NavItem>
            <NavItem
                        className={undefined}
                        tag="li"
            >
                        <NavLink
                                    activeClassName="active"
                                    aria-current="page"
                                    href="/myReports"
                                    to="/myReports"
                        >
                                    <div
                                                className={undefined}
                                    />
                                    MES REPORTINGS
                        </NavLink>
            </NavItem>
            <NavItem
                        className={undefined}
                        tag="li"
            >
                        <NavLink
                                    activeClassName="active"
                                    aria-current="page"
                                    href="/myCollaborativeSpace"
                                    to="/myCollaborativeSpace"
                        >
                                    <div
                                                className={undefined}
                                    />
                                    MON ESPACE COLLABORATIF
                        </NavLink>
            </NavItem>
</Nav>,
        ],
        "className": undefined,
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <Media
              className={undefined}
              object={true}
              src="atome.svg"
/>,
            "href": "https://www.paprec.com/undefined",
            "target": "_blank",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "className": undefined,
              "object": true,
              "src": "atome.svg",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          "type": "a",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "children": Array [
              <NavItem
                className={undefined}
                tag="li"
>
                <NavLink
                                activeClassName="active"
                                aria-current="page"
                                href="/home"
                                to="/home"
                >
                                <div
                                                className={undefined}
                                />
                                ACCUEIL
                </NavLink>
</NavItem>,
              <NavItem
                className={undefined}
                tag="li"
>
                <NavLink
                                activeClassName="active"
                                aria-current="page"
                                href="/myContracts"
                                to="/myContracts"
                >
                                <div
                                                className={undefined}
                                />
                                MES CONTRATS
                </NavLink>
</NavItem>,
              <NavItem
                className={undefined}
                tag="li"
>
                <NavLink
                                activeClassName="active"
                                aria-current="page"
                                href="/myFiles"
                                to="/myFiles"
                >
                                <div
                                                className={undefined}
                                />
                                MES DOCUMENTS 
                                 EN LIGNE
                </NavLink>
</NavItem>,
              <NavItem
                className={undefined}
                tag="li"
>
                <NavLink
                                activeClassName="active"
                                aria-current="page"
                                href="/myRemovals"
                                to="/myRemovals"
                >
                                <div
                                                className={undefined}
                                />
                                MES ENLÈVEMENTS
                </NavLink>
</NavItem>,
              <NavItem
                className={undefined}
                tag="li"
>
                <NavLink
                                activeClassName="active"
                                aria-current="page"
                                href="/myReports"
                                to="/myReports"
                >
                                <div
                                                className={undefined}
                                />
                                MES REPORTINGS
                </NavLink>
</NavItem>,
              <NavItem
                className={undefined}
                tag="li"
>
                <NavLink
                                activeClassName="active"
                                aria-current="page"
                                href="/myCollaborativeSpace"
                                to="/myCollaborativeSpace"
                >
                                <div
                                                className={undefined}
                                />
                                MON ESPACE COLLABORATIF
                </NavLink>
</NavItem>,
            ],
            "className": undefined,
            "tag": "ul",
            "vertical": false,
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "children": <NavLink
                  activeClassName="active"
                  aria-current="page"
                  href="/home"
                  to="/home"
>
                  <div
                                    className={undefined}
                  />
                  ACCUEIL
</NavLink>,
                "className": undefined,
                "tag": "li",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "activeClassName": "active",
                  "aria-current": "page",
                  "children": Array [
                    <div
                      className={undefined}
/>,
                    "ACCUEIL",
                  ],
                  "href": "/home",
                  "to": "/home",
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "className": undefined,
                    },
                    "ref": null,
                    "rendered": null,
                    "type": "div",
                  },
                  "ACCUEIL",
                ],
                "type": [Function],
              },
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "children": <NavLink
                  activeClassName="active"
                  aria-current="page"
                  href="/myContracts"
                  to="/myContracts"
>
                  <div
                                    className={undefined}
                  />
                  MES CONTRATS
</NavLink>,
                "className": undefined,
                "tag": "li",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "activeClassName": "active",
                  "aria-current": "page",
                  "children": Array [
                    <div
                      className={undefined}
/>,
                    "MES CONTRATS",
                  ],
                  "href": "/myContracts",
                  "to": "/myContracts",
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "className": undefined,
                    },
                    "ref": null,
                    "rendered": null,
                    "type": "div",
                  },
                  "MES CONTRATS",
                ],
                "type": [Function],
              },
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "children": <NavLink
                  activeClassName="active"
                  aria-current="page"
                  href="/myFiles"
                  to="/myFiles"
>
                  <div
                                    className={undefined}
                  />
                  MES DOCUMENTS 
                   EN LIGNE
</NavLink>,
                "className": undefined,
                "tag": "li",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "activeClassName": "active",
                  "aria-current": "page",
                  "children": Array [
                    <div
                      className={undefined}
/>,
                    "MES DOCUMENTS 
 EN LIGNE",
                  ],
                  "href": "/myFiles",
                  "to": "/myFiles",
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "className": undefined,
                    },
                    "ref": null,
                    "rendered": null,
                    "type": "div",
                  },
                  "MES DOCUMENTS 
 EN LIGNE",
                ],
                "type": [Function],
              },
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "children": <NavLink
                  activeClassName="active"
                  aria-current="page"
                  href="/myRemovals"
                  to="/myRemovals"
>
                  <div
                                    className={undefined}
                  />
                  MES ENLÈVEMENTS
</NavLink>,
                "className": undefined,
                "tag": "li",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "activeClassName": "active",
                  "aria-current": "page",
                  "children": Array [
                    <div
                      className={undefined}
/>,
                    "MES ENLÈVEMENTS",
                  ],
                  "href": "/myRemovals",
                  "to": "/myRemovals",
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "className": undefined,
                    },
                    "ref": null,
                    "rendered": null,
                    "type": "div",
                  },
                  "MES ENLÈVEMENTS",
                ],
                "type": [Function],
              },
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "children": <NavLink
                  activeClassName="active"
                  aria-current="page"
                  href="/myReports"
                  to="/myReports"
>
                  <div
                                    className={undefined}
                  />
                  MES REPORTINGS
</NavLink>,
                "className": undefined,
                "tag": "li",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "activeClassName": "active",
                  "aria-current": "page",
                  "children": Array [
                    <div
                      className={undefined}
/>,
                    "MES REPORTINGS",
                  ],
                  "href": "/myReports",
                  "to": "/myReports",
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "className": undefined,
                    },
                    "ref": null,
                    "rendered": null,
                    "type": "div",
                  },
                  "MES REPORTINGS",
                ],
                "type": [Function],
              },
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "children": <NavLink
                  activeClassName="active"
                  aria-current="page"
                  href="/myCollaborativeSpace"
                  to="/myCollaborativeSpace"
>
                  <div
                                    className={undefined}
                  />
                  MON ESPACE COLLABORATIF
</NavLink>,
                "className": undefined,
                "tag": "li",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "activeClassName": "active",
                  "aria-current": "page",
                  "children": Array [
                    <div
                      className={undefined}
/>,
                    "MON ESPACE COLLABORATIF",
                  ],
                  "href": "/myCollaborativeSpace",
                  "to": "/myCollaborativeSpace",
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "className": undefined,
                    },
                    "ref": null,
                    "rendered": null,
                    "type": "div",
                  },
                  "MON ESPACE COLLABORATIF",
                ],
                "type": [Function],
              },
              "type": [Function],
            },
          ],
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromError": true,
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    Symbol(enzyme.__providerValues__): undefined,
  },
  Symbol(enzyme.__providerValues__): Array [],
  Symbol(enzyme.__childContext__): null,
}
`;
